packages/patternfly-4/react-core/src/components/DataList/DataList.md

```diff
@@ -96,15 +98,35 @@ import {
 class CheckboxActionDataList extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { isOpen: false };
+    this.state = { isOpen1: false, isOpen2: false };
```

Comment:
is `isOpen3` missing here?

Reply:
That's odd I thought I added it. Thanks for the double check.